autogtp/main.cpp

```diff
@@ -106,7 +116,26 @@ bool fetch_best_network(QTextStream& cerr, QString& netname) {
     return true;
 }
 
-bool upload_data(QTextStream& cerr, const QString& netname, QString sgf_output_path) {
+QString extract_netname(const QString& fileName) {
```

Comment:
One reason why I suggested dumping old games if the network name changes was that it would avoid the need for this code. That is, you could just remember the network name your played games were done with, rather than having to read and parse it back.